server/cluster/coordinator_test.go

```diff
@@ -286,10 +286,11 @@ func prepare(setCfg func(*config.ScheduleConfig), setTc func(*testCluster), run
 	}
 	tc := newTestCluster(ctx, opt)
 	hbStreams := hbstream.NewTestHeartbeatStreams(ctx, tc.getClusterID(), tc, true /* need to run */)
+	co := newCoordinator(ctx, tc.RaftCluster, hbStreams)
+	tc.coordinator = co
```

Comment:
Maybe we can merge these two lines and use `run(tc.coordinator)` later?

Reply:
There is not much difference here.